sycl/test-e2e/Matrix/joint_matrix_bfloat16_impl.hpp

```diff
@@ -135,6 +135,7 @@ int main() {
 
     if (combinations[i].nsize == 8) { // architecture::intel_gpu_dg2*
       test<bfloat16, float, /*TM*/ 8, /*TN*/ 8, /*TK*/ 16>();
+      test<bfloat16, float, /*TM*/ 32, /*TN*/ 32, /*TK*/ 16>();
```

Comment:
Please add the same change to the element wise operations tests like https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/Matrix/element_wise_all_ops_impl.hpp#L239
Make sure joint_matrix_apply works with this new combination

Reply:
I have added the test :) Thanks!